README.md

```diff
@@ -85,3 +85,5 @@ python
 ```
 
 To use GPUs in `waveorder` set ```use_gpu=True``` when initializing the simulator and reconstructor classes.
```

Comment:
This is the CuPy toggle and not relevant to the PyTorch implementation.

Reply:
I don't know if it still works or not, but we should consider removing any mention of GPU in user-facing documentation before we have a fully working torch port.

Reply:
I agree, I just removed this section from the README.